lib/caasp.pm

```diff
@@ -153,9 +153,6 @@ sub process_reboot {
         elsif (match_has_tag 'grub2') {
             send_key 'ret';
         }
-        else {
-            die 'Kubic did not manage to reboot properly';
```

Comment:
Why should this code be unreachable, isn't an explicit else branch always a good idea?

Reply:
Wouldn't the assert_screen basically finish the test if the  needles with the corresponding task are not found?, therefore making that branch, indeed, as he's covering the only two possible scenarios?.

Reply:
In this matter I plan to create a ticket, that we can put the comment explaining check for all `assert_*` methods and not only `assert_script_run`. And then we can force it so no new assert is added without such message. It should explain review, as sometimes we do complex parcing and it's not clear what it is all about.